### DISCUSSION TITLE
Buttons for stepping additional to the sliders

### DISCUSSION THREAD

**Discussion by osi1880vr** [hlky/stable-diffusion-webui#880] [Feature suggestions]
Hello,

I like the ideas of all the sliders and the ease of use they bring.
But sometimes it would be great to have more precission while changing the values.

If I like to move a slider just a single step I sometimes have a hard time to get it done.
It would be great to have little buttons for +1 Step and - 1 Step.
Simple up and down arrows would be sufficent.

I think this would be a great addon to the sliders.
Alternative would be a textfield where I just enter the value I like to have.

Im trying to get it done myself but I start learning the frondendstuff just now, So my guess is you might be far more quick =)

Thanks a lot.

If I get it done first I will raise a pull request ;)

**Comment by thedyze** [comment]
You can use the left and right arrow keys to move a selected slider one step at the time
